Update index.mdx (#5394)
EnterpriseDB/docs: product_docs/docs/biganimal/release/getting_started/identity_provider/index.mdx

When using your own identity provider, you add users to BigAnimal by adding them to the designated group in your identity provider. Once you've logged into BigAnimal using your own identity provider, you can set up your cloud service provider in the BigAnimal portal to complete onboarding. 

> If you're using the BigAnimal portal as your identity provider, you can also invite users that have an EDB account by selecting **Invite New User** on the Users page. After providing their EDB account email and their role, you can send them an invitation link. Please pay attention that the invitation link will be expired in 48 hours. For more information on roles, see [Managing portal access](/biganimal/release/administering_cluster/01_portal_access/).

Reviewer: Let's add some information on what would happen if the invitation expires. How about this? 
```suggestion
If you're using the BigAnimal portal as your identity provider, you can also invite users that have an EDB account by selecting **Invite New User** on the Users page. After providing their EDB account email and their role, you can send them an invitation link. Ensure the user accepts the invitation within 48 hours, or you'll have to send a new invitation. 

For more information on roles, see [Managing portal access](/biganimal/release/administering_cluster/01_portal_access/).
```
